HADotNet.Core/HADotNet.Core.csproj

```diff
@@ -32,7 +32,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="RestSharp" Version="106.10.0" />
+    <PackageReference Include="RestSharp" Version="106.10.1" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="RestSharp.Serializers.NewtonsoftJson" Version="106.10.1" />

```

Comment:
Not needed since we are not using `Client.UseNewtonsoftJson();`.

Reply:
Yes, this is not needed any more.